kuma/settings/common.py

```diff
@@ -622,9 +622,7 @@ def _get_locales():
             'extensions': [
                 'jinja2.ext.do',
                 'jinja2.ext.loopcontrols',
-                'jinja2.ext.with_',
```

Comment:
In TravisCI, the locales are built with the current Docker image, which means it is breaking on this change that requires the updated requirements are used. I'm going to add a new commit to change the ``locale`` build to act more like the ``py27``, so the updated requirements will be used.